themes/Valendas/lxqt-panel.qss

```diff
@@ -176,6 +176,119 @@ QMenu QToolButton {
     color: white;
 }
 
+/*
+ * Fancy menu plugin
+ */
+#FancyMenuPlugin {
+    padding: 0px;
+}
+
+#FancyMenu {
+    qproperty-icon: url(mainmenu.svg);
+    background-color: rgba(0, 0, 0, 10%);
+    margin-right: 5px;
+    color: #ebebeb;
+}
+
+#FancyMenu:hover {
+    background-color: rgba(5, 0, 0, 45%);
+}
+
+#FancyMenu:pressed {
+    padding: 1px;
+}
+LXQtFancyMenuWindow {
+   background-color: #3b3b3b; /* disabled transparency because buggy */
+}
+
+#FancyMenu QLineEdit {
+    margin: 0;
+    padding: 6px;
+    color: white;
+    border-radius: 0px;
+    background: #444444;
+}
+
+#FancyMenu #AppView {
+    border-left: 1px solid white;
+    border-right: 1px solid white;
+    background-color: #444444;
+    margin: 1px;
+}
```

Comment:
If you don't want top and bottom borders, you need to add this (otherwise, they might be taken from Qt styles like Kvantum and the result will be ugly):

```css
    border-top: 1px solid transparent;
    border-bottom: 1px solid transparent;
```

If not, then

```css
#FancyMenu #AppView {
    border: 1px solid white;
    background-color: #444444;
    margin: 1px;
}
```

Reply:
Done, in fact it wasn't working and I forgot about, I forgot that it needs all 4. 
White was ugly, they are greyish. Found another issue, it needs `outline:transparent` too.

And another big issue: with "windows" widget style all scrollbars are ugly in all themes.